### PR TITLE
ref: multi-target netcore 3.0

### PR DESCRIPTION
**Pull request by bruno-garcia**
Addresses #303 but means targeting `netcoreapp3.0` which requires some thought and a major bump.

It was supposed to be a draft PR*

### CODE REVIEW

File: src/Sentry.AspNetCore/HostingEnvironmentExtensions.cs [hunk 1]
```diff
@@ -0,0 +1,18 @@
...
+
+namespace Sentry.AspNetCore
+{
+    internal static class HostingEnvironmentExtensions
+    {
+        public static string RootPath(this IWebHostEnvironment env) => env.
+#if NETSTANDARD2_0
+            WebRootPath;
+#else
+            ContentRootPath;
```
Comment (khellang) [inline src/Sentry.AspNetCore/HostingEnvironmentExtensions.cs]: These are not the same paths... 🙈

Comment (bruno-garcia) [reply src/Sentry.AspNetCore/HostingEnvironmentExtensions.cs]: Lol, good catch, thanks @khellang
`WebRootPath` isn't there on the 3.0.0 version (it's the right namespace so I don't know what's wrong).

Comment (khellang) [reply src/Sentry.AspNetCore/HostingEnvironmentExtensions.cs]: You could try to do something [like this] (https://github.com/khellang/Middleware/blob/fc75a0402df3a941c7fd464af4a8c116257e3e4e/src/ProblemDetails/ProblemDetailsMiddleware.cs#L14-L18) to reduce churn 😊

Comment (bruno-garcia) [reply src/Sentry.AspNetCore/HostingEnvironmentExtensions.cs]: That looks better. Thanks for the tip.

**Pull request merged**